README.md

```diff
@@ -33,6 +33,22 @@ In the meantime, contributions to other areas of Service Fabric are welcome on a
 
 For more information on how this process works and how to contribute, provide feedback, and log bugs, please see [Contributing.md](CONTRIBUTING.md).
 
+## Build Requirements
+The requirements below are based off running a clean build using ninja, with the command
+
+$ runbuild.sh –c –n
```

Comment:
Ninja might actually slow down a clean build, it might be worth looking at the time it takes without ninja.  On a D64s in azure it took me 35 minutes to do a clean build